packages/experimental/Spinner/src/Spinner.styling.tsx

```diff
@@ -0,0 +1,49 @@
+import type { UseStylingOptions } from '@fluentui-react-native/framework';
+import { buildProps } from '@fluentui-react-native/framework';
+import type { SpinnerProps, SpinnerSlotProps, SpinnerTokens } from './Spinner.types';
+import { spinnerName } from './Spinner.types';
+import { defaultSpinnerTokens } from './SpinnerTokens';
+
+export const diameterSizeMap: { [key: string]: number } = {
```

Comment:
Can the type be updated to:
`[key: SpinnerSize]: number` if the key is always expected to be size?

Reply:
If we convert it results in : export const diameterSizeMap: { [key in SpinnerSize]: number } = {
  'xx-small': 12,
  'x-small': 16,
  medium: 24,
  large: 32,
  'x-large': 36,
  tiny: 0,
  small: 0,
  huge: 0
}; 

where we need to add tiny , small , huge which is not part of Android/Mobile sizing.

Reply:
Added that.